src/content/docs/fr/guides/troubleshooting.mdx

```diff
@@ -1,23 +1,68 @@
 ---
 title: Dépannage
 description: Besoin d'aide ? Bloqué sur quelque chose ? Ce guide est fait pour vous.
+i18nReady: true
 ---
 
 Astro fournit plusieurs outils différents pour vous aider à dépanner et à déboguer votre code.
 
-## Erreurs courantes
+## Trucs et astuces
 
-Voici une liste de quelques messages d'erreurs courants que vous pourriez rencontrer dans votre terminal, ce qu'ils signifient et ce que vous pouvez faire pour les corriger.
+### Débogage avec `console.log()`
+
+`console.log()` est un moyen simple mais populaire de déboguer votre code Astro. Lorsque vous écrivez votre appel à `console.log`, il déterminera où votre sortie de débogage sera affichée.
+
+```astro
+---
+console.log('Salut! Je suis le serveur. Ceci est affiché dans le terminal où Astro est exécuté.');
+---
+
+<script>
+console.log('Salut! Je suis le client. Ceci est affiché dans la console de développement du navigateur.');
+</script>
+```
+
+Un appel à `console.log()` dans le Frontmatter d'Astro sera toujours sorti dans le **terminal** qui exécute le processus d'Astro. Car Astro s'exécute sur le serveur, et jamais dans le navigateur.
+
+Le code qui est écrit ou importé dans un `<script>` d'Astro sera exécuté dans le navigateur. Toutes les sorties de débogage ou autres appels à `console.log()` seront affichés dans votre navigateur.
+
+### Débogage des composants de Framework
+
+[Les composants de Framework](/fr/core-concepts/framework-components/) (comme React et Svelte) sont uniques : ils sont rendus sur le serveur par défaut, ce qui signifie que les sorties de débogage seront visibles dans le terminal. Cependant, ils peuvent également être hydratés pour le navigateur, ce qui peut aussi entraîner que vos logs de débogage apparaissent également dans le navigateur.
+
+Cela peut être utile pour déboguer les différences entre la sortie SSR et les composants hydratés dans le navigateur.
+
+### Le composant `<Debug />` d'Astro
 
-### Transform failed with X error
+Pour vous aider à déboguer vos composants Astro, Astro propose un composant [`<Debug />`](/fr/reference/api-reference/#debug-) qui affiche directement une valeur dans votre template HTML de composant. Cela est utile pour un débogage rapide dans le navigateur sans avoir à faire les aller-retour entre votre terminal et votre navigateur.
```

Comment:
I think its better to say "des allers-retours" instead of "les aller-retour"
```suggestion
Pour vous aider à déboguer vos composants Astro, Astro propose un composant [`<Debug />`](/fr/reference/api-reference/#debug-) qui affiche directement une valeur dans votre template HTML de composant. Cela est utile pour un débogage rapide dans le navigateur sans avoir à faire des allers-retours entre votre terminal et votre navigateur.
```

Reply:
Good point, ty